packages/app-project/src/screens/ClassifyPage/components/ClassifierWrapper/ClassifierWrapper.js

```diff
@@ -147,6 +148,12 @@ ClassifierWrapper.propTypes = {
   subjectID: string,
   /** optional subject set ID (from the page URL.) */
   subjectSetID: string,
+  /** Current logged-in user */
+  user: shape({
+    error: string
```

Comment:
Getting console "Warning: Failed prop type: Invalid prop `user.error` of type `object` supplied to `ClassifierWrapper`, expected `string`."